### ISSUE TITLE
Issue starting the app

### ISSUE THREAD

**Issue by lyonelf** [Xstoudi/Palisades#2]
When i want to launch Palisade, it need .net 6

After download and install, same message appear.

After restart KO too.

**Comment by Xstoudi** [comment]
Can you provid more informations? Windows version, dotnet version? Screenshot of the message would be cool.
Thanks a lot!

**Comment by lyonelf** [comment]
I'm in w11.
1st launch, the redirect trought .nt 6 download page was ok.
Installation ok.

After that, same message:

![2023-01-05_09h56_17](https://user-images.githubusercontent.com/34097487/210740664-3c028709-c22a-4e3a-882f-da5efc1fbb33.png)

**Comment by Xstoudi** [comment]
I'm not able to test on Win11 yet. If someone that can wants to take a look it would be cool.

**Comment by krimsonkharne** [comment]
On Win11 here and it worked for me, although the link to the missing frameworks didn't automatically open after clicking on "Yes".
However after downloading .net framework 6 palisades does now work for me.
@lyonelf after installing .net 6 sdk, can you run the command "dotnet" in cmd - does that give any errors?

**Comment by krimsonkharne** [comment]
Hm, actually, scratch that - Palisades opens, but dragging and dropping files into a palisade does nothing.

**Comment by nabemoon** [comment]
Hello, I had a similar problem, when I tried to run the program after installation it showed a message saying that I needed to download a .NET version to execute, but it did not lead me anywhere. I mannualy upadted the .net 6 version and installed the 8 version as well. Now Palisades runs and I can create 1 palisade "folder", but I cannot edit anything and the right mouse click does not work. I will attach some images and if theres any more help with info let me know, I really liked the program and would love to help even though I don't program in C#. 
!
![Imagem do WhatsApp de 2024-09-18 à(s) 17 27 36_2950f4a8](https://github.com/user-attachments/assets/324025c7-9097-4b90-ba0b-64613987590b)
![Imagem do WhatsApp de 2024-09-18 à(s) 17 28 34_48c779ef](https://github.com/user-attachments/assets/b3b8f032-c4ca-4cb2-b735-5d4789da274a)
![Imagem do WhatsApp de 2024-09-18 à(s) 17 40 52_beebc64e](https://github.com/user-attachments/assets/a5cae0a6-f447-4471-aee1-facceecee2a4)

**Comment by marshenmanhunter** [comment]
yeah i have the same issue here. how did you guys update the .net files?

**Comment by wallofchaos** [comment]
Same error. and i got irritated with it. because I installed various versions of .net and it still doesnt work. 

So I tried UNINSTALLING. There is NO uninstaller.. Why????
So i deleted the files and it still starts with windows with thsi dumb error. Now i dont hvae the files and no way to uninstall and I get this Lovely error everytime I reboot my PC. LOVELY!
